.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
Change this to .idea/ so that it can match the full .idea directory. Otherwise, it's interpreted as a file